api/lib/infrastructure/repositories/sessions/session-repository.js

```diff
@@ -195,6 +195,16 @@ module.exports = {
       .first();
     return Boolean(result);
   },
+
+  async hasNoStartedCertification(sessionId) {
+    const result = await knex
+      .select(1)
+      .from('sessions')
+      .innerJoin('certification-courses', 'certification-courses.sessionId', 'sessions.id')
+      .where('sessions.id', sessionId)
+      .first();
+    return Boolean(!result);
```

Comment:
`!` inverse la valeur en transformant au prealable en boolean. Boolean(!1) correspond à Boolean(true), le "Boolean" n'est donc plus pertinent (oui c'est subtil 😉)

 ```suggestion
     return !Boolean(result);
```

Reply:
Menteur :p

![image](https://user-images.githubusercontent.com/37305474/192489859-42341786-4063-4efd-850f-1768f1ead968.png)